release notes/upcoming.md

```diff
@@ -2,11 +2,9 @@ TODO: Intro
 
 ## New Features!
 
-### Category: Title (#533)
+### HTTP: request body compression (#988)
 
-Description of feature.
-
-**Docs**: [Title](http://k6.readme.io/docs/TODO)
+Now all http methods have an additional param called `compression` that will make k6 to compress the body before sending it. It will also correctly set both `Content-Encoding` and `Content-Length`, except if they were not set previous to that. The current supported algorithms are `deflate` and `gzip` and any combination of the two separated by a comma (`,`).
```

Comment:
"that will make k6 to compress" should either be "that will make k6 compress" or "that will cause k6 to compress"

"except if they were not set previous to that" doesn't feel right and isn't very clear, I suggest something like "unless they were manually set in the request `headers` by the user"...

Reply:
fixed